webpack/assets/javascripts/react_app/components/common/Fill/FillActions.js

```diff
@@ -0,0 +1,24 @@
+import { REGISTER_FILL, REMOVE_FILLED_COMPONENT } from './FillConstants';
+import { add, remove } from './ExtendableRegistery';
```

Comment:
[Google](https://www.google.com/search?client=firefox-b&q=Dictionary#dobs=registry) knows only the version without 'e', could you rename please?